dist/chart/templates/crd/servicemonitor_crd.yaml

```diff
@@ -0,0 +1,1236 @@
+{{- if .Values.crd.enable }}
+# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.80.0/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
```

Comment:
Same question as for the kustomize, shouldn't this come from the `prometheus-operator` helm chart?

Reply:
Not sure about it as it's generated by `make helm`